src/index.js

```diff
@@ -793,6 +791,21 @@ export default class AuthnWidget {
     if (this.store.state.showCaptcha && this.grecaptcha) {
       this.grecaptcha.render(this.captchaDivId);
     }
+
+    let alreadyAutofocus = false;
+    let inputElements = document.querySelectorAll("input:not(:disabled)")
+    inputElements.forEach(ele => {
```

Comment:
NIT: I think we can query on attributes as well and avoid the for loop. Something like `'input:not(:disabled)[autofocus]'` should allow for it.